src/styles/blocks/main/_star.scss

```diff
@@ -0,0 +1,12 @@
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background: no-repeat 50% url(../images/star.svg);
+}
+
+@for $index from 1 through $active-stars {
```

Comment:
Same comment. You should reuse stars block from the previous task
![image](https://user-images.githubusercontent.com/98764641/203539825-515bb46f-de96-4648-a531-808437c8c101.png)